towhee/utils/hub_utils.py

```diff
@@ -197,6 +197,27 @@ def delete_token(self, token_id: int, password: str) -> None:
         except HTTPError as e:
             raise e
 
+
+
+    def basic_create(self, password, repo_class):
+        data = {
+            'auto_init': True,
+            'default_branch': 'main',
+            'description': repo_class,
+            'name': self._repo,
+            'private': False,
+            'template': False,
+            'trust_model': 'default',
+            'type': repo_class
+        }
+        url = self._root + '/api/v1/user/repos'
+        try:
+            r = requests.post(url, data=data, auth=HTTPBasicAuth(self._author, password))
+            r.raise_for_status()
+        except HTTPError as e:
+            raise e
+
```

Comment:
Why do we need this?

Reply:
What is the difference between `create` and `basic_create` ? `basic_create` does not require a token?

Reply:
basic create doesnt use tokens, rather the login info. I believe there is an authorization error when you try to push to a repo that was made with tokens.

Reply:
Is that for #542